### PR TITLE
Update flashing.md

### PR DESCRIPTION
**Pull request by Bartosz-Slowik**
Extended Flashing chapter. Included DFU as requested.

### CODE REVIEW

File: docs/getting-started/src/flashing.md [hunk 1]
```diff
@@ -22,8 +22,49 @@ STM32f411CEU6 microcontroller doesn't come with a programmer/debugger on board.
...
-Connect the ST-Link and the MCU into your PC using USB cables and run the following command from command-line:
+Connect the ST-Link and the MCU into your PC using USB cables and run the following command from command-line inside cansat repository:
 
 ```
-cargo xtask embed cansat-stm32f4
+cargo xtask embed
 ```
+
+## Using standalone ST-LINK
+1. Connect the vires according to schematic below:
```
Comment (0xf4lc0n) [inline docs/getting-started/src/flashing.md:32]: Connect the wires according to the diagram below:

File: docs/getting-started/src/flashing.md [hunk 2]
```diff
@@ -22,8 +22,49 @@ STM32f411CEU6 microcontroller doesn't come with a programmer/debugger on board.
...
+<p>
+<img src="https://github.com/grupacosmo/cansat/assets/92270288/cedabacc-694a-4266-8d2e-87d94bb18612">
+</p>
+
+2. Connect the ST-Link and the MCU into your PC using USB cables and run the following command from command-line inside cansat repository:
```
Comment (0xf4lc0n) [inline docs/getting-started/src/flashing.md:37]: Connect the ST-Link and the MCU to your PC using USB cables. Then in the cansat directory run the following command from command-line

File: docs/getting-started/src/flashing.md [hunk 3]
```diff
@@ -22,8 +22,49 @@ STM32f411CEU6 microcontroller doesn't come with a programmer/debugger on board.
...
+
+
+**Info: libusb-win32 did not work when testing**
+
+Run:
+```
+cargo install cargo-dfu
+```
+Connect your board to USB, hold BOOT0 button, press NRST button and release BOOT0 button. This should run the MCU in bootloader mode.
+**The blue LED will not flash when inside this mode. Red LED will be on constantly**
```
Comment (0xf4lc0n) [inline docs/getting-started/src/flashing.md:55]: The blue LED will not flash in this mode. The red LED will be on constantly

File: docs/getting-started/src/flashing.md [hunk 4]
```diff
@@ -22,8 +22,49 @@ STM32f411CEU6 microcontroller doesn't come with a programmer/debugger on board.
...
+```
+cd crates/cansat-stm32f4
+```
+and run 
+```
+cargo dfu
+```
+
+### Troubleshooting
+If your device is not properly recognized by your system, try to disconnect all serial peripherals and repeat.
```
Comment (0xf4lc0n) [inline docs/getting-started/src/flashing.md:67]: If your device is not recognized correctly by your system, try disconnecting all serial peripherals and repeat the steps above.

**Review by 0xf4lc0n — CHANGES_REQUESTED** [review]
There are a few typos and stylistic errors worth correcting.

**Comment by Bartosz-Slowik** [comment]
Applied all the suggestions

**Review by 0xf4lc0n — APPROVED** [review]

**Pull request merged**